docs/installing-open-mpi/configure-cli-options/networking.rst

```diff
@@ -68,12 +68,12 @@ can be used with ``configure``:
   Set configuration values for Portals 4
 
 * ``--with-psm=<directory>``:
-  Specify the directory where the QLogic InfiniPath / Intel True Scale
+  Specify the directory where the QLogic InfiniPath / Cornelis Netorks True Scale
```

Comment:
The PSM(v1) MTL was removed in commit 0348d14ff3c0; this bullet for `--with-psm` and the next bullet for `--with-psm-libdir` should be removed.